src/alerts/version-polling.ts

```diff
@@ -13,7 +15,7 @@ export const checkVersion = withErrorIgnoring(async (): Promise<void> => {
   if (latestVersion !== currentVersion) {
     const badVersions = await getBadVersions();
     if (badVersions.includes(currentVersion)) {
-      versionStore.update((value) => ({ ...value, isUpdateRequired: true }));
+      versionStore.update((value) => ({ ...value, updateRequiredBy: Date.now() + FORCED_UPDATE_DELAY }));
```

Comment:
Why not make it 15 minutes from now? I'm wondering what the impact will be if they reload a page, for instance. Won't that cause them to reload sooner than expected?

Reply:
Sure, we can increase this delay. Though I don't _think_ there's much of anything in Terra where the user would have a large batch of unsaved input.

Reply:
Increased to 10 minutes in https://github.com/DataBiosphere/terra-ui/pull/4559/commits/69cd7ad7b84d13f16e8fc89ebf8ee9bd4608907c.